frontend/src/components/RegistrationForm/RegistrationForm.js

```diff
@@ -0,0 +1,235 @@
+import React, { useState } from 'react';
+import { 
+  Button,
+  CardContent,
+  IconButton,
+  InputAdornment,
+  Grid,
+  Paper,
+  TextField,
+  Typography
+} from '@material-ui/core';
+import visibilityIcon from '../../assets/generic/visibility.svg';
+import visibilityOffIcon from '../../assets/generic/visibility-off.svg';
+import productIcon from "../../assets/generic/product-icon.svg"
+import { emailRegex, passwordRegex, nameRegex } from "../../constants/regex"
+
+export default function RegistrationForm(props) {
+  const [firstName, setFirstName] = useState("")
+  const [lastName, setLastName] = useState("")
+  const [email, setEmail] = useState("")
+  const [password, setPassword] = useState("")
+  const [confirmPassword, setConfirmPassword] = useState("")
+  const [showPassword, setShowPassword] = useState(false)
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false)
+  const [showError, setShowError] = useState({ 
+    errorFirstName: false,
+    errorLastName: false,
+    errorEmail: false,
+    errorPassword: false,
+    errorConfirmPassword: false
+  })
```

Comment:
ugh, where are all the semicolons, bro? Can you run prettier + eslint manually please?

Reply:
Ugh, do we really need semicolons?

Reply:
Ran Prettier in logout-button branch